vic/vic_run/include/vic_run.h

```diff
@@ -186,41 +210,51 @@ int get_volume(lake_con_struct, double, double *);
 double hiTinhib(double);
 int ice_melt(double, double, double *, double, snow_data_struct *,
              lake_var_struct *, double, double, double, double, double, double,
+             double,
              double, double, double, double, double, double, double, double,
-             double, double *, double *, double *, double *, double *, double *,
-             double *, double *, double *);
+             double *,
+             double *, double *, double *, double *, double *, double *,
+             double *,
+             double *);
 double IceEnergyBalance(double, va_list);
 void iceform(double *, double *, double, double, double *, int, double, double,
-             double, double *, double *, double *, double *, double);
+             double, double *, double *, double *, double *,
+             double);
 void icerad(double, double, double, double *, double *, double *);
 void initialize_lake(lake_var_struct *, lake_con_struct, soil_con_struct *,
-                     cell_data_struct *, bool);
+                     cell_data_struct *,
+                     bool);
 int lakeice(double, double, double, double, double, double *, double, double *,
-            double *, double, double);
+            double *, double,
+            double);
 void latent_heat_from_snow(double, double, double, double, double, double,
                            double, double *, double *, double *, double *,
                            double *);
 void latsens(double, double, double, double, double, double, double, double,
-             double *, double *, double);
+             double *, double *,
+             double);
 double linear_interp(double, double, double, double, double);
 double lkdrag(double, double, double, double, double);
 void malloc_2d_double(size_t *shape, double ***array);
 void malloc_3d_double(size_t *shape, double ****array);
 void MassRelease(double *, double *, double *, double *);
 double maximum_unfrozen_water(double, double, double, double);
 double new_snow_density(double);
-int newt_raph(void (*vecfunc)(double *, double *, int, int,
-                              ...), double *, int);
+int newt_raph(void (*vecfunc)(double *, double *, int, int, ...), double *, int);
 double penman(double, double, double, double, double, double, double);
 void photosynth(char, double, double, double, double, double, double, double,
                 double, double, char *, double *, double *, double *, double *,
                 double *);
 void polint(double xa[], double ya[], int n, double x, double *y, double *dy);
-void prepare_full_energy(int, all_vars_struct *, soil_con_struct *, double *,
+void prepare_full_energy(cell_data_struct *, energy_bal_struct *,
+                         soil_con_struct *, double *,
                          double *);
```

Comment:
The change to prepare_full_energy() is the only non-format modification.  The rest of the changes in vic_run.h came from running uncrustify.